cmd/builder/internal/builder/config.go

```diff
@@ -39,6 +39,7 @@ type Config struct {
 	Processors   []Module     `mapstructure:"processors"`
 	Connectors   []Module     `mapstructure:"connectors"`
 	Providers    *[]Module    `mapstructure:"providers"`
+	Converters   *[]Module    `mapstructure:"converters"`
```

Comment:
Why pointer?

Reply:
I was following the model used to add providers, but I'm not aware of the context as to why that decision was made

Reply:
Will change that soon. See also https://github.com/open-telemetry/opentelemetry-collector/pull/11596

Reply:
Now, see how I use it for providers https://github.com/open-telemetry/opentelemetry-collector/pull/11566

Reply:
thanks this makes sense, I'll update the PR and tests accordingly